index.html

```diff
@@ -4415,22 +4421,42 @@ <h3>JSON-LD</h3>
 
         <ul>
           <li>
-Use of JSON-LD keywords in the `@context` value that globally modify document
-value processing, such as global settings of `@base`
+In-line declaration of JSON-LD keywords in the `@context` value that globally
+modify document term and value processing, such as setting `@base` or `@vocab`
           </li>
           <li>
-In-line declaration of JSON-LD contexts in the `@context` property.
+Use of JSON-LD contexts that override declarations in previous contexts, such as
+resetting `@vocab`
+          </li>
+          <li>
+In-line declaration of JSON-LD contexts in the `@context` property
           </li>
           <li>
 Use of full URLs for JSON-LD terms and types (e.g.,
 `https://www.w3.org/2018/credentials#VerifiableCredential` or
 `https://vocab.example/myvocab#SomeNewType`) instead of the short forms of
 any such values (e.g., `VerifiableCredential` or `SomeNewType`) that are
-either explicitly defined as JSON-LD `@context` mappings (e.g.,
-`https://www.w3.org/ns/credentials/v2`) or are implicitly defined via the
-`@vocab` feature that applies to all undefined terms.
+explicitly defined as JSON-LD `@context` mappings (e.g.,
+`https://www.w3.org/ns/credentials/v2`)
           </li>
-       </ul>
+        </ul>
+
+        <p>
+While this specification cautions against the use of `@vocab`, there are
+legitimate uses of the feature, such as to ease experimentation, development,
+and localized deployment. If an application developer wants to use `@vocab` in
+production — even though strongly advised against — they are urged to understand that
+any use of `@vocab` will disable reporting of "undefined term" errors, and
```

Comment:
```suggestion
production, which is advised against to reduce term collisions and leverage the benefits
of semantic interoperability, they are urged to understand that any use of `@vocab` will
disable reporting of "undefined term" errors, and
```

tried to maintain line spacing but may need another reformat